src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Response/PatchArtistResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace CryptoAxus.Application.Features.Artist.PatchArtistUserWalletAddress.Response;
+
+public class PatchArtistResponse : BaseResponse<object>
```

Comment:
`BaseResponse<object>` should not be returned here. It should be `BaseResponse<ArtistDto>`

Reply:
Resolved